.github/workflows/ci_rust.yml

```diff
@@ -97,6 +131,17 @@ jobs:
           cd ../../..
           ldd ${{env.STANDARD_PATH}}/target/debug/integration | grep libs2n.so
 
+          # Run tests with the external build
```

Comment:
Do we have any validation that s2n_tls_external_build isn't set when it's not an external build?

Reply:
I added a new test for this!